src/main/docs/guide/azureCosmosClient.adoc

```diff
@@ -0,0 +1,5 @@
+https://docs.microsoft.com/en-us/azure/cosmos-db/introduction/[Azure Cosmos DB] is a fully managed NoSQL database for modern app development. Single-digit millisecond response times, and automatic and instant scalability, guarantee speed at any scale.
+
+To add support for Azure Cosmos DB to an existing project, add the following dependency to your build.
+dependency:micronaut-azure-cosmos[groupId="io.micronaut.azure"]
```

Comment:
Perhaps it is worth mentioning the beans that will be available for injecting: builder, sync client, async client, etc.

Reply:
It is kind of mentioned [here](https://github.com/micronaut-projects/micronaut-azure/pull/374/files#diff-82cd9a6805668f8ab3cda1994c2fa4dd68a8d129b0b16cf867b364989f882bb8)

Reply:
Oh you are right!. But knowing that the builder is also available is helpful IMO

Reply:
Sure, added that info as well.